apps/resources-wcm/src/main/webapp/skin/less/ecms/portlets/attachments/attachmentsApp.less

```diff
@@ -21,6 +21,10 @@
       font-size: 41px;
     }
 
+    .file-type-icon {
+      flex: 0 0 75%;
```

Comment:
please avoid addin new class for this property that you can make with CSS Classes helpers.